app/res/layout/blue_outlined_button.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Button xmlns:android="http://schemas.android.com/apk/res/android"
+<com.google.android.material.button.MaterialButton xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
can we have a screenshot of this button in PR description as well, it's the 'Clear' button that appears on configuing a [multi-select question in the form](https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2213544088/List+of+Question+Types+in+CommCare#Multiple-Choice)

Reply:
![clear](https://github.com/user-attachments/assets/002509ff-f5aa-431a-b9c8-d8ff4a08ef38)